charts/rd-professional-api/values.preview.template.yaml

```diff
@@ -6,7 +6,7 @@ java:
     POSTGRES_NAME: "{{ .Values.postgresql.postgresqlDatabase}}"
     POSTGRES_USERNAME: "{{ .Values.postgresql.postgresqlUsername}}"
     POSTGRES_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
-
+    USER_PROFILE_URL: http://rd-user-profile-api-pr-404.service.core-compute-preview.internal
```

Comment:
I think we need to make sure to delete this before merging master.As it is pointing to pr-404.

Reply:
Yes ..once UP code merge to master i will remove this .